corehq/pillows/sms.py

```diff
@@ -42,3 +55,22 @@ def calc_meta(cls):
         so we just do a hash of the "prototype" instead to determined md5
         """
         return cls.calc_mapping_hash({"es_meta": cls.es_meta, "mapping": cls.default_mapping})
+
+
+def get_sql_sms_pillow():
```

Comment:
You'll need to add an ID to this now that https://github.com/dimagi/commcare-hq/pull/10816 has been merged.